### DISCUSSION TITLE
Issue with Sending Text Messages via API

### DISCUSSION THREAD

**Discussion by YMaldi** [devlikeapro/waha#292] [Q&A]
Hey,
I'm attempting to send a text message using the following API call:
POST /api/sendText
{
  "session": "$sessionName",
  "chatId": "[$phoneNumber@c.us](mailto:$phoneNumber@c.us)",
  "text": "Hi there!"
}
My session is already connected, and I've scanned the QR code. I begin by sending 'startTyping', then I wait for a delay before sending 'stopTyping'. After another delay, I attempt to send the text message
Initially, everything worked well, and I was able to send messages successfully. However, suddenly it stopped working, and now it only returns :
 "statusCode": 500,
    "timestamp": "2024-04-10T21:10:19.515Z",
    "exception": {
        "stack": "Error: Connection Closed\n    at sendRawMessage (/app/node_modules/@adiwajshing/baileys/lib/Socket/socket.js:54:19)\n    at sendNode (/app/node_modules/@adiwajshing/baileys/lib/Socket/socket.js:73:16)\n    at query (/app/node_modules/@adiwajshing/baileys/lib/Socket/socket.js:137:15)\n    at getUSyncDevices (/app/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:166:30)\n    at /app/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:357:53\n    at Object.transaction (/app/node_modules/@adiwajshing/baileys/lib/Utils/auth-utils.js:136:32)\n    at relayMessage (/app/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:279:30)\n    at Object.sendMessage (/app/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:652:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/node_modules/@nestjs/core/router/router-execution-context.js:46:28",
        "message": "Connection Closed",
        "data": null,
        "isBoom": true,
        "isServer": false,
        "output": {},
        "name": "Error"
    },
    "request": {
       only data for my query here
        },
        "query": {}
    },
    "version": {
        "version": "2024.4.1",
        "engine": "NOWEB",
        "tier": "PLUS",
        "browser": "/usr/bin/chromium"
    }
}
Could you please assist me with resolving this issue?

**Comment by allburov** [comment]
@YMaldi if you restart the container with empty `.sessions` folder and start a new session, scan QR - it doesn't help, does it?
